provider/maas/environ.go

```diff
@@ -1052,6 +1062,55 @@ func (environ *maasEnviron) StartInstance(args environs.StartInstanceParams) (
 	}, nil
 }
 
+func instanceConfiguredInterfaceNames(usingMAAS2 bool, inst instance.Instance, subnetsMap map[string]network.Id) ([]string, error) {
+	var (
+		interfaces []network.InterfaceInfo
+		err        error
+	)
+	if !usingMAAS2 {
```

Comment:
We should do this as a separate change; there are many other places that pass in / use the result of usingMAAS2().